services/alerta/service.go

```diff
@@ -185,6 +192,10 @@ func (s *Service) preparePost(token, tokenPrefix, resource, event, environment,
 		origin = c.Origin
 	}
 
+	if timeout == toml.Duration(0) {
```

Comment:
This can be simplified as `if timeout == 0 {` Go will infer the type of the constant numeric literal.